testsuite/MDAnalysisTests/core/test_atomselections.py

```diff
@@ -1019,3 +1019,13 @@ def test_missing_icodes_range_VE(self, u):
         u = make_Universe(('resids',))
         with pytest.raises(ValueError):
             u.select_atoms('resid 10A-12')
+
+class TestArbitraryAtomGroupSelection:
+    def test_arbitrary_atom_group_raises_error(self):
+        u = mda.Universe(GRO)
```

Comment:
these universes can be replaced by a `make_Universe()` equivalent to save on some file I/O